controls/radgridview/export/how-to/warning-message-excel.md

```diff
@@ -26,6 +26,6 @@ The alert is a new security feature in Excel 2007 called Extension Hardening, wh
 
 __SOLUTION__
 
-You can [ export to ExcelML format]({%slug gridview-export%}) with XML extension. In this way you will not get such warning.
+You can [ export to ExcelML format]({%slug gridview-export%}) with XML extension. In such case you will avoid the warning.
```

Comment:
Latest versions of Excel are not very fond of the XML extension. Consider changing this to XLSX or at least add it as another option.